pueue-*: add page (#5285)
tldr-pages/tldr: pages/common/pueue-add.md

# pueue add

> Enqueue a task for execution.

- Add any command to the default queue:

`pueue add {{command}}`

- Pass flags or arguments to command when adding:

`pueue add -- {{command --Arg -f}}`

- Add command but do not start it if it's the first in a queue:

`pueue add --stashed -- rsync -az /local/directory/ /remote/directory/`

> - Add command to a [g]roup and start it [i]mmmediately, see "tldr pueue-group" to operate groups:

Author: > P.S. FYI, you can use tldr command subcommand (without the dash), too ;)

I didn't know the client could do that. Cool!

# pueue add

> Enqueue a task for execution.
> More information: <https://github.com/Nukesor/pueue>.

> - Add any command to the default queue:

Reviewer: I think we should remove both this and the next example and replace them with a combination, something like this:
```markdown
- Add a command to the queue in the default group:

`pueue add -- {{command}} {{args}}`
```

> 

Reviewer: I think we should remove both this and the next example and replace them with a combination, something like this:
```markdown
- Add a command to the queue in the default group:

`pueue add -- {{command}} {{args}}`
```

`pueue add {{command}}`

> - Pass flags or arguments to command when adding:

Reviewer 2: Perhaps this will read better?
```suggestion
- Pass a list of flags or arguments to a command when enqueuing:
```

`pueue add -- {{command --arg -f}}`

> - Add command but do not start it if it's the first in a queue:

Reviewer 2: Perhaps this will read better?
```suggestion
- Add a command but do not start it if it's the first in a queue:
```
Same goes for the others here.

> `pueue add --stashed -- rsync --archive --compress {{/local/directory}} {{/remote/directory}}`

Reviewer: ```suggestion
`pueue add --stashed -- {{rsync --archive --compress /local/directory /remote/directory}}`
```

> - Add command to a group and start it immmediately, see `pueue group` to manage groups:

Reviewer: ```suggestion
- Add command to a group and start it immediately, see `pueue group` to manage groups:
```

> - Add command to a group and start it immediately, see `pueue group` to manage groups:

Reviewer 3: ```suggestion
- Add a command to a group and start it immediately, see `pueue group` to manage groups:
```

`pueue add --immediate --group "{{CPU_intensive}}" -- {{ffmpeg -i input.mp4 frame_%d.png}}`

> - Add command and start it after commands 9 and 12 finish successfully:

Reviewer 3: ```suggestion
- Add a command and start it after commands 9 and 12 finish successfully:
```

`pueue add --after {{9}} {{12}} --group "{{torrents}}" -- {{transmission-cli torrent_file.torrent}}`

> - Add command with a label after some delay has passed, see `pueue enqueue` for valid datetime formats:

Reviewer 3: ```suggestion
- Add a command with a label after some delay has passed, see `pueue enqueue` for valid datetime formats:
```
